client/src/components/application/DotsStepper/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import MobileStepper from '@material-ui/core/MobileStepper';
+import PropTypes from 'prop-types';
+import stepperStyle from './style';
+import registration from './registrationSteps.json';
+import Typography from '../../common/Typography';
+import Card from '../../common/card';
+import Button from '../../common/Button';
+
+const { primaryHeader, Info, stepHeader, steps } = registration;
```

Comment:
we will get steps from database & pass them as props.
```suggestion
const { primaryHeader, Info, stepHeader } = registration;
```